bin/jurassic-tube-setup.sh

```diff
@@ -3,29 +3,25 @@
 # Exit if any command fails.
 set -e
 
-echo "Checking if ${PWD}/docker/bin/jt directory exists..."
+# Define Jurassic Tube directory using bin directory
+JT_DIR="${PWD}/bin/jurassictube"
 
-if [ -d "${PWD}/docker/bin/jt" ]; then
-    echo "${PWD}/docker/bin/jt already exists."
-else
-    echo "Creating ${PWD}/docker/bin/jt directory..."
-    mkdir -p "${PWD}/docker/bin/jt"
-fi
-
-echo "Downloading the latest version of the installer script..."
-echo 
+echo "Setting up Standalone Jurassic Tube..."
 
-# Download the installer (if it's not already present):
-if [ ! -f "${PWD}/docker/bin/jt/installer.sh" ]; then
-    # Download the installer script:
-    curl "https://jurassic.tube/get-installer.php?env=wcpay" -o ${PWD}/docker/bin/jt/installer.sh && chmod +x ${PWD}/docker/bin/jt/installer.sh
+# Create Jurassic Tube directory if it doesn't exist
+if [ ! -d "$JT_DIR" ]; then
+    echo "Creating Jurassic Tube directory at $JT_DIR..."
+    mkdir -p "$JT_DIR"
 fi
 
-echo "Running the installation script..."
-echo 
+# Download the installer only if it's not already present
+if [ ! -f "$JT_DIR/installer.sh" ]; then
+    echo "Downloading the standalone installer..."
+    curl "https://jurassic.tube/installer-standalone.sh" -o "$JT_DIR/installer.sh" && chmod +x "$JT_DIR/installer.sh"
```

Comment:
We no longer need to provide the environment value, freeing us from coupling it to the product and avoiding the addition of new environments to JT with each new product.